idemp-rest/WEB-INF/src/org/bandahealth/idempiere/rest/service/db/ChargeDBService.java

```diff
@@ -138,7 +143,10 @@ protected Charge createInstanceWithDefaultFields(MCharge_BH instance) {
 
 	@Override
 	protected Charge createInstanceWithAllFields(MCharge_BH instance) {
-		return new Charge(instance);
+		MElementValue account =  accountDBService.getEntityByIdFromDB(instance.getC_ElementValue_ID());
```

Comment:
This doesn't batch the calls. Can you update so that calls are batched?

Reply:
ok